MonticelloTonel-FileSystem.package/.squot-contents

```diff
@@ -1,4 +1,4 @@
 SquotTrackedObjectMetadata {
-	#serializer : #SquotCypressCodeSerializer,
-	#objectClassName : #PackageInfo
+	#objectClassName : #PackageInfo,
+	#serializer : #SquotCypressCodeSerializer
```

Comment:
Looks like I should investigate why this order changes... But maybe it is only to recent changes in Squeak's Dictionary.